packages/core/admin/ee/admin/pages/SettingsPage/pages/ReviewWorkflows/pages/ListView/ListView.js

```diff
@@ -0,0 +1,166 @@
+      }
+    }
+  }
+`;
+
+export function ReviewWorkflowsListView() {
+  const { formatMessage } = useIntl();
+  const { push } = useHistory();
+  const { workflows: workflowsData } = useReviewWorkflows();
+  const dispatch = useDispatch();
+  const { status, serverState } = useSelector((state) => state?.[REDUX_NAMESPACE] ?? initialState);
+
+  useInjectReducer(REDUX_NAMESPACE, reducer);
+
+  useEffect(() => {
+    dispatch(setWorkflows({ status: workflowsData.status, data: workflowsData.data }));
+  }, [workflowsData.status, workflowsData.data, dispatch]);
+
+  return (
+    <CheckPagePermissions permissions={adminPermissions.settings['review-workflows'].main}>
+      <Layout>
+        <SettingsPageTitle
+          name={formatMessage({
+            id: 'Settings.review-workflows.page.title',
+            defaultMessage: 'Review Workflows',
+          })}
+        />
+        <Main tabIndex={-1}>
+          <HeaderLayout
+            subtitle={formatMessage({
+              id: 'Settings.review-workflows.list.page.subtitle',
+              defaultMessage:
+                'Manage content review stages and collaborate during content creation from draft to publication',
+            })}
+            title={formatMessage({
+              id: 'Settings.review-workflows.list.page.title',
+              defaultMessage: 'Review Workflows',
+            })}
+          />
+
+          <ContentLayout>
+            {status === 'loading' && (
+              <Loader>
+                {formatMessage({
+                  id: 'Settings.review-workflows.page.list.isLoading',
+                  defaultMessage: 'Workflows are loading',
+                })}
+              </Loader>
+            )}
+
+            <Table colCount={3} rowCount={1}>
+              <Thead>
+                <Tr>
+                  <Th>
+                    <Typography variant="sigma">
+                      {formatMessage({
+                        id: 'Settings.review-workflows.list.page.list.column.name.title',
+                        defaultMessage: 'Name',
+                      })}
+                    </Typography>
+                  </Th>
+                  <Th>
+                    <Typography variant="sigma">
+                      {formatMessage({
+                        id: 'Settings.review-workflows.list.page.list.column.stages.title',
+                        defaultMessage: 'Stages',
+                      })}
+                    </Typography>
+                  </Th>
+                  <Th>
+                    <VisuallyHidden>
+                      {formatMessage({
+                        id: 'Settings.review-workflows.list.page.list.column.actions.title',
+                        defaultMessage: 'Actions',
+                      })}
+                    </VisuallyHidden>
+                  </Th>
+                </Tr>
+              </Thead>
+
+              <Tbody>
+                {serverState?.workflows.map((workflow) => (
+                  <Tr
```

Comment:
Think this probably needs `cursor: pointer`?

Reply:
We don't do that in other places, but I will double-check!

Reply:
Turns out: 8425df9f5463d9c2effd8e427e10ec63f3b1adfe 🤦🏼